app/models/organization.rb

```diff
@@ -0,0 +1,10 @@
+class Organization < Publisher
+  default_scope { where(role: Publisher::ORGANIZATION) }
+
+  validate :no_email
+  has_many :memberships
+
+  def no_email
```

Comment:
Do we intend to find these by name? since there are no emails?

Curious what the workflow is for creating these organizations